biome.jsonc

```diff
@@ -26,7 +23,6 @@
     "rules": {
       "recommended": true,
```

Comment:
can this be upstreamed?

Reply:
Already part of `@chainsafe/biomejs-config/recommended`, removed from here.